copyright

```diff
@@ -386,6 +386,21 @@ Copyright: Copyright Mike Koenig
 License: CC-BY-SA-3.0
 Comment: Taken from http://soundbible.com/1467-Grenade-Explosion.html
 
+Files: sounds/missile hit.wav
+Copyright: Copyright "Nbs Dark"
+License: Public Domain
+Comment: Taken from https://freesound.org/people/Nbs%20Dark/sounds/94185/
+
+Files: sounds/meteor.wav
+Copyright: Copyright "18hiltc"
+License: CC-BY-3.0
+Comment: Taken from https://freesound.org/people/18hiltc/sounds/202725/
+
+Files: sounds/sidewinder.wav
+Copyright: Copyright "Robinhood76"
+License: CC-BY-NC-3.0
```

Comment:
Non-commercial licenses aren't compatible with the Debian software guidelines, so this sound can't be used in Endless Sky.

Reply:
Fixed; I found another sound.